src/components/Tabs/Tabs.tsx

```diff
@@ -164,17 +163,13 @@ export const Tabs = ({
    * Autogenerate ids on tabs if not defined.
    */
   useEffect(() => {
-    setId(
-      tabs().map((tab) => (tab.props.id ? tab.props.id : uidSeed(`${tab}-id`))),
-    );
+    setId(tabs().map((tab) => (tab.props.id ? tab.props.id : getUID(tab))));
```

Comment:
Would a better type on `idVar` / `setId` have prevented this issue?

I'm also surprised TS let us use `setId` with `any[]` as the implicit type 🤔

Reply:
I think the typing improvement should be made on the `getUID` side instead (or better docs on their side) -- `getUID` was still returning a string (that happened to be something like `"1val-[object Object]-id"`, which is pretty meaningless), so better typing of `setId` wouldn't have caught it

Reply:
Oh, it's because `strict` mode is off on the `next` branch.

@anniehu4, do you know why that is? My recommendation is to turn that on. Probably would've prevented this issue.

Reply:
although yes, agree that there are an unfortunate number of `any` types in this file 😬

Reply:
yup, there are a couple of things we had to turn off in `next` because the existing BigMedium code had a bunch of violations, and it wasn't clear which components we needed to touch yet. We've been tracking this particular issue: https://app.shortcut.com/czi-edu/story/187478/re-enable-strict-true-in-tsconfig-json?cf_workflow=500033078&ct_workflow=all&groupBy=workflowState

Reply:
Ah gotcha, so either way it returns a string. We still probably always want `strict: true`.